pyro/contrib/mue/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright Contributors to the Pyro project.
+# SPDX-License-Identifier: Apache-2.0
+"""
+The :mod:`pyro.contrib.mue` module provides tools for working with mutational
+emission (MuE) distributions.
+See Weinstein and Marks (2020),
+https://www.biorxiv.org/content/10.1101/2020.07.31.231381v1.
+Primary developer is Eli N. Weinstein (https://eweinstein.github.io/).
+"""
```

Comment:
Note you can either keep this text here, or move it to docs/source/contrib.mue.rst. From that .rst file you should use `automodule` to include this module. See other .rst files there for example.